documentation/src/main/resources/theme/default-theme.yml

```diff
@@ -0,0 +1,271 @@
+font:
```

Comment:
Because, if I want to update it and base it on a new default-theme.yml file, it's going to be messy. This way, it's really easy to keep track of the changes.

I did that a lot in ${previous-job} and it's really more practical like that.

I'll add the reference to the GitHub file at the top of our theme but I prefer to keep the original file too to be on the safe side.